partners_api/facebook_ads.cpp

```diff
@@ -3,6 +3,27 @@
 #include "indexer/classificator.hpp"
 #include "indexer/feature_data.hpp"
 
+namespace
+{
+using namespace facebook;
+
+template <typename IterType>
+IterType FindType(feature::TypesHolder const & types, IterType first, IterType last)
```

Comment:
As you use return value only for check - exists|non-exists, why not to return bool?
I.e.:
```
bool HasTypeInRange(feature::TypesHolder const & types, It begin, It end);
```
?